From 70d46d5cd8bac0de99fed21ee2247ec74b03991b Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Mon, 5 Oct 2015 14:15:57 +0200
Subject: build: Require the newest gdk-pixbuf

Current development version of gdk-pixbuf is required for the
GDK_PIXBUF_ERROR_INCOMPLETE_ANIMATION error.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index b75e203..f49b423 100644
--- a/configure.ac
+++ b/configure.ac
@@ -113,7 +113,7 @@ GLIB_TESTS
 dnl If you add a version number here, you *must* add an AC_SUBST line for
 dnl it too, or it will never make it into the spec file!
 
-GDK_PIXBUF_REQUIRED=2.21.3
+GDK_PIXBUF_REQUIRED=2.33.0
 GTK_REQUIRED=3.3.6
 GLIB_REQUIRED=2.44.0
 XRANDR_REQUIRED=1.3
-- 
cgit v0.12

